src/display/annotation_layer.js

```diff
@@ -33,6 +33,8 @@ import {
 import { AnnotationStorage } from "./annotation_storage.js";
 import { ColorConverters } from "../shared/scripting_utils.js";
 
+const DEFAULT_TAB_INDEX = 1000;
```

Comment:
Nit: you could define this in a common file to avoid defining it twice

Reply:
The value is not the same for annotations and for xfa elements to avoid any conflicts (it should be possible to have some annotations and some xfa elements with XFAF).